src/components/forms/ProjectGrantsForm.tsx

```diff
@@ -174,6 +176,13 @@ export const ProjectGrantsForm: FC = () => {
                     </PageText>
                   </Box>
                 )}
+                {errors?.firstName?.type === 'validate' && (
+                  <Box mt={1}>
+                    <PageText as='small' fontSize='helpText' color='red.500'>
+                      First name contain a URL.
```

Comment:
For consistency within the error messages and more clarity, we should rephrase it as 'First name cannot contain a URL'